packages/flutter/lib/src/rendering/error.dart

```diff
@@ -50,6 +51,9 @@ class RenderErrorBox extends RenderBox {
 
   /// The message to attempt to display at paint time.
   final String message;
+  final double systemTopPadding = Zone.current['systemTopPadding'] == null
```

Comment:
Yes, but I thinking we'd make this change in `dart:ui` so that the `ui.window` property was retrieved from the Zone.

```dart
final Window _window = new Window._();
Window get window => Zone.current[Window] ?? _window;
```